src/app/components/LiveLabel/index.stories.tsx

```diff
@@ -36,7 +36,7 @@ const Component = ({
 };
 
 export default {
-  title: 'New Components/Live Label',
+  title: 'New Components/Live Label Promo',
```

Comment:
Shall we remove 'Promo' from the name?
```suggestion
  title: 'New Components/Live Label',
```